_includes/badge-ui-toolkit.html

```diff
@@ -0,0 +1,3 @@
+<span class="badge badge-ui-toolkit">
+    JS<span class="sr-only"> is required for use of this pattern</span>
```

Comment:
You could also use a JS badge icon such as http://www.friersonswebsite.com/images/javascript_icon.png if it is freely available.